Objects/index.js

```diff
@@ -0,0 +1,189 @@
+//Створи об'єкт, що описує автомобіль (виробник, модель, рік випуску,
+//середня швидкість, обсяг паливного баку, середня витрата палива на
+//100 км., водії) Метод, який виводить на екран інформацію про автомобіль.
+//Додавання ім’я водія у список
+//Перевірка водія на наявність його ім’я у списку
+//Підрахунок необхідного часу та кількості палива для подолання
+//переданої відстані з середньою швидкістю. Враховуй, що через кожні 4
+//години дороги водієві необхідно робити перерву на 1 годину.
+const car = {
+  prod: "VW",
+  model: "Passat",
+  speed: 120,
+  fuelConsumption: 5.5,
+  driverName: [],
+  showInfo() {
+    console.log("Information about the car:");
+
+    console.log(
+      `Producer: ${this.prod}, model is ${this.model}, average speed is ${this.speed}, average fuel consumption is ${this.fuelConsumption}, driverName is: ${this.driverName}`
+    );
+  },
+  addName(name) {
+    this.driverName.indexOf(name) === -1
```

Comment:
if the name is in the list should be separate method so there are should be two methods 
addName 
checkIfDriverNameExistst

also array is called driverName and method addName  so there are should be more clear naming either (names: [], addName: () => {} ) or (driverNames: [] and addDriverName: () => {})